### PR TITLE
controller initial work

### PR DESCRIPTION
**Pull request by EvenSol**

### CODE REVIEW

File: src/main/java/neqsim/processSimulation/processEquipment/ProcessEquipmentBaseClass.java [hunk 1]
```diff
@@ -285,4 +286,18 @@ public boolean equals(Object obj) {
...
+     * @return the runTransient
+     */
+    public boolean isRunTransient() {
+      return runTransient;
+    }
+
+    /**
+     * @param runTransient the runTransient to set
+     */
+    public void setRunTransient(boolean runTransient) {
```
Comment (asmfstatoil) [inline src/main/java/neqsim/processSimulation/processEquipment/ProcessEquipmentBaseClass.java:300]: this should not be public

File: src/main/java/neqsim/processSimulation/processEquipment/ProcessEquipmentInterface.java [hunk 2]
```diff
@@ -186,4 +186,11 @@
     /** {@inheritDoc} */
     @Override
     public int hashCode();
+    
+    public boolean isRunTransient();
+
+    /**
+     * @param runTransient the runTransient to set
+     */
+    public void setRunTransient(boolean runTransient);
```
Comment (asmfstatoil) [inline src/main/java/neqsim/processSimulation/processEquipment/ProcessEquipmentInterface.java:195]: should not be public

File: src/main/java/neqsim/processSimulation/processEquipment/ProcessEquipmentBaseClass.java [hunk 3]
```diff
@@ -31,6 +31,7 @@ public abstract class ProcessEquipmentBaseClass extends SimulationBaseClass
     ControllerDeviceInterface flowValveController = null;
     public boolean hasController = false;
     private String specification = "TP";
+    private boolean runTransient = false;
```
Comment (asmfstatoil) [inline src/main/java/neqsim/processSimulation/processEquipment/ProcessEquipmentBaseClass.java:34]: should not be named the same as the function runTransient.

Comment (EvenSol) [reply src/main/java/neqsim/processSimulation/processEquipment/ProcessEquipmentBaseClass.java:34]: Good point. Will change that.

**Comment by EvenSol** [comment]
if (getCalculateSteadyState()) 

if (getCalculateSteadyState()) is true - then the unit operation should be calculated as steady stat using normal run operation. For me it seems as the opposite is done (that transient is run if getCalculateSteadyState() is true.

**Comment by asmfstatoil** [comment]
That is a bug/typo.

**Pull request merged**